app/api/v2/views/redflag_views.py

```diff
@@ -123,3 +118,22 @@ def patch(self, id):
             return {"data": [{"New Comment": data['comment'],
                               "message": "Updated comment successfully"}]}, 200
         return {"data": [{"message": " Incident not found"}]}, 404
+
+
+class MyStatusRecords(Resource, IncidentsModel):
+    """ Edit the comment of a specific intervention record. """
+
+    def __init__(self):
+        self.db = IncidentsModel()
+
+    @jwt_required
+    def patch(self, id):
+        """ Admin makes changes to status"""
+        fetch_by_id = self.db.get_incident_by_id(id=id)
+        data = request.get_json(force=True)
+
+        if fetch_by_id:
+            self.db.update_status(data['status'], id)
+            return {"data": [{"New status": data['status'],
+                              "message": "Updated status successfully"}]}, 200
+        return {"data": [{"message": " Status Incident not found"}]}, 404
```

Comment:
return only one dictionary

Reply:
Thank you for the feedback, I have resolved this.